src/site/content/en/blog/monitor-total-page-memory-usage/index.md

```diff
@@ -97,68 +88,34 @@ are less useful. Example use cases:
+result of the API is highly implementation dependent because browsers have
 different ways of representing objects in memory and different ways of
-estimating the memory usage. Browsers may exclude some memory regions from
+estimating memory usage. Browsers may exclude some memory regions from
 accounting if proper accounting is too expensive or infeasible. Thus, results
 cannot be compared across browsers. It is only meaningful to compare the
 results for the same browser.
 
-## Current status {: #status }
-
-<div>
-<table>
-    <thead>
-      <tr>
-        <th>Step</th>
-        <th>Status</th>
-      </tr>
-    </thead>
-    <tbody>
-      <tr>
-        <td>1. Create explainer</td>
-        <td><a href="https://github.com/WICG/performance-measure-memory">Complete</a></td>
-      </tr>
-      <tr>
-        <td>2. Create initial draft of specification</td>
-        <td><a href="https://wicg.github.io/performance-measure-memory/">Complete</a></td>
-      </tr>
-      <tr>
-        <td>3. Gather feedback and iterate design</td>
-        <td><a href="#feedback">In Progress</a></td>
-      </tr>
-      <tr>
-        <td>4. Origin trial </td>
-        <td><a href="https://developers.chrome.com/origintrials/#/view_trial/1281274093986906113">Complete</a></td>
-      </tr>
-      <tr>
-        <td>5. Launch</td>
-        <td>Enabled by default in Chrome 89</td>
-      </tr>
-    </tbody>
-</table>
-</div>
-
 ## Using `performance.measureUserAgentSpecificMemory()` {: use }
 
-### Enabling via about://flags
-
-To experiment with `performance.measureUserAgentSpecificMemory()` without an origin trial
-token, enable the `#experimental-web-platform-features` flag in `about://flags`.
-
 ### Feature detection
 
-The `performance.measureUserAgentSpecificMemory()` function may fail with a
-[SecurityError][security-error] if the execution environment does not fulfil
+The `performance.measureUserAgentSpecificMemory` function will be unavailable or may
+fail with a [SecurityError][security-error] if the execution environment does not fulfil
 the security requirements for preventing cross-origin information leaks.
-During the origin trial in Chrome, the API requires that [Site
-Isolation][site-isolation] is enabled. When the API ships, it will rely on
-[cross-origin isolation][cross-origin-isolation]. A web page can opt-in to
-cross-origin isolation by setting [COOP+COEP headers][coop-coep].
+It relies on [cross-origin isolation][cross-origin-isolation], which a web page can activate
+by setting [COOP+COEP headers](/coop-coep/).
+
+Support can be detected at runtime:
 
 ```javascript
-if (performance.measureUserAgentSpecificMemory) {
+if (!window.crossOriginIsolated) {
+  console.log('performance.measureUserAgentSpecificMemory() is only available in cross-origin-isolated pages');
+} else if (!performance.measureUserAgentSpecificMemory) {
+  console.log('performance.measureUserAgentSpecificMemory() is not available in this browser');
+} else {
```

Comment:
This is admittedly a little awkward, but it's important to be aware of the three possible states, not just if the API is available or not. Happy to adjust further.